target/sim/vsim/start.cheshire_soc.tcl

```diff
@@ -12,7 +12,9 @@ set TESTBENCH tb_cheshire_soc
 # Set voptargs only if not already set to make overridable.
 # Default on fast simulation flags.
 if { ![info exists VOPTARGS] } {
-    set VOPTARGS "-O5 +acc=p+tb_cheshire_soc. +noacc=p+cheshire_soc. +acc=r+stream_xbar"
+    # set VOPTARGS "-O5 +acc=p+tb_cheshire_soc. +noacc=p+cheshire_soc. +acc=r+stream_xbar"
+    # No optimisations
+    set VOPTARGS "+acc"
```

Comment:
I think we can keep the upstream version here.

Reply:
ok